javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,96 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector("section.products ul");
+console.log(productsUl);
+
+function renderProducts(products) {
+  productsUl.innerHTML = "";
+  products.forEach((product) => {
+    const li = document.createElement("li");
+
+    let shipsToHTML = "";
+    product.shipsTo.forEach(
+      (country) => (shipsToHTML += `<li>${country}</li>`)
+    );
+
+    li.innerHTML = `
+            <ul>
+                <li>${product.name}</li>
+                <li>${product.price}</li>
+                <li>${product.rating}</li>
+                <ul class="ships-to">${shipsToHTML}</ul>
+            </ul>
```

Comment:
Unordered list contains elements of the same type. You could use just divs here or make it a [table](https://www.w3schools.com/html/html_tables.asp).

Reply:
I think it is  inside homework, I just copied it. It is not mine. Any way thank you 👍